Tests/test_integration.py

```diff
@@ -190,6 +190,29 @@ def __disable_integrations_instances(client, module_instances):
             print_error(pformat(res))
 
 
+def __enable_integrations_instances(client, module_instances):
+    for configured_instance in module_instances:
+        # tested with POSTMAN, this is the minimum required fields for the request.
+        module_instance = {
+            key: configured_instance[key] for key in ['id', 'brand', 'name', 'data', 'isIntegrationScript', ]
+        }
+        module_instance['enable'] = "true"
+        module_instance['version'] = -1
+
+        try:
+            res = demisto_client.generic_request_func(self=client, method='PUT',
+                                                      path='/settings/integration',
+                                                      body=module_instance)
+        except ApiException as conn_err:
+            print_error(
+                'Failed to enable integration instance, error trying to communicate with demisto '
+                'server: {} '.format(
+                    conn_err))
+
+        if res[1] != 200:
+            print_error('enable instance failed with status code ' + str(res[1]) + '\n' + pformat(res))
```

Comment:
You probably didn't notice but `__enable_integrations_instances` isn't actually used anywhere currently. It's essentially copied wholesale from the `__disable_integrations_instances` function above it so the behavior is identical to there. I added it in anticipation of when `test_content.py` is refactored and will use the instances already configured for the (Test Button) tests instead of configuring new instances for executing the test playbooks. As it's more of a placeholder right now, I think whoever works on the `test_content.py` refactor should update this function, as necessary, to suit how they plan on using it. But if you don't agree, I can change it to what you suggested. Let me know.